### PR TITLE
Updated simplify and detail for dflclient

### PR DESCRIPTION
**Pull request by Nilanshrajput**
## Description
Corrected DataCentricFLClient serialization. the method, the previous method has bugs and didn't serialise it correctly.
More specifically running the below code raises an error and connection breaks but the method in this PR will solves that
```
x = torch.tensor([1,2,3,4,5]) 
x = x.send(bob) 
x = x.share(bob,alice)
```
This comes after testing and discussion from @IonesioJunior and me.

## Affected Dependencies
List any dependencies that are required for this change.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Review by IonesioJunior — APPROVED** [review]
LGTM! Thank you @Nilanshrajput!
